open_prices/proofs/models.py

```diff
@@ -563,6 +564,13 @@ def save(self, *args, **kwargs):
         self.full_clean()
         super().save(*args, **kwargs)
 
+    @property
+    def image_path(self):
+        from open_prices.proofs.utils import get_price_tag_image_path
+
+        image_path = get_price_tag_image_path(self.id)
+        return image_path if os.path.exists(image_path) else None
```

Comment:
not a fan of having to import + check if the file exists + ternary for each returned PriceTag 👀 
(vs storing the info in a dedicated model field)